drivers/nvdimm/pmem.c

```diff
@@ -31,6 +33,24 @@
 #include "pfn.h"
 #include "nd.h"
 
```

Comment:
There were many conflicts in this file. I did not patch anything here but mostly updates were made here from upstream and patch was not applying cleanly.